dlt/common/configuration/specs/config_providers_context.py

```diff
@@ -40,7 +39,51 @@ def add_provider(self, provider: ConfigProvider) -> None:
 
     @staticmethod
     def initial_providers() -> List[ConfigProvider]:
-        return [EnvironProvider(), SecretsTomlProvider(add_global_config=True), ConfigTomlProvider(add_global_config=True)]
+        providers = [
+            EnvironProvider(),
+            SecretsTomlProvider(add_global_config=True),
+            ConfigTomlProvider(add_global_config=True)
+        ]
+        providers += _extra_providers()
```

Comment:
@rudolfix I wanted to decouple airflow provider from ConfigProvidersContext in a simples way possible. 
Later, if needed, we can implement this with a proper plugin-based architecture.